isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -2311,12 +2375,36 @@ namespace Isis {
       if (!temp.expanded().startsWith("/")) {
         temp = FileName(m_labelFileName->path() + "/" + temp.original());
       }
-
-      initCoreFromLabel(Pvl(temp.toString()));
+      try {
+        initCoreFromLabel(Pvl(temp.toString()));
+      }
+      catch (IException &e) {
+        initCoreFromGdal(temp.toString());
+      }
```

Comment:
If there is a specific catch we want here we should probably do that instead.

Reply:
Is there a way to delineate ISIS exceptions? It's an object not found exception from the PvlObject but it just throws an IException